analyzer/src/funTest/kotlin/GradleTest.kt

```diff
@@ -88,5 +107,48 @@ class GradleTest : StringSpec() {
             result!!.errors should beEmpty()
             yamlMapper.writeValueAsString(result) shouldBe expectedResult
         }
+
+        "Is compatible with Gradle >= 3.3" {
+            val gradleVersions = table(
+                    headers("version", "resultsFileSuffix"),
+                    row("3.3", "-3.3"),
+                    row("3.4", ""),
+                    row("3.4.1", ""),
+                    row("3.5", ""),
+                    row("3.5.1", ""),
+                    row("4.0", ""),
+                    row("4.0.1", ""),
+                    row("4.0.2", ""),
+                    row("4.1", ""),
+                    row("4.2", ""),
+                    row("4.2.1", ""),
+                    row("4.3", ""),
+                    row("4.3.1", ""),
+                    row("4.4", ""),
+                    row("4.4.1", ""),
+                    row("4.5", ""),
+                    row("4.5.1", "")
+            )
+
+            forAll(gradleVersions) { version, resultsFileSuffix ->
+                gradleWrapper(version)
+
+                val packageFile = File(projectDir, "app/build.gradle")
+                val expectedResult = patchExpectedResult("gradle-expected-output-app$resultsFileSuffix.yml")
+
+                val result = Gradle.create().resolveDependencies(listOf(packageFile))[packageFile]
+
+                result shouldNotBe null
+                result!!.errors should beEmpty()
+                yamlMapper.writeValueAsString(result) shouldBe expectedResult
+            }
+        }.config(tags = setOf(ExpensiveTag))
+    }
+
+    private fun gradleWrapper(version: String) {
+        println("Installing Gradle wrapper version $version.")
+        val gradleWrapper = if (OS.isWindows) Gradle.wrapper else "./${Gradle.wrapper}"
```

Comment:
@sschuberth : Do you have an idea why this does not work on Windows?

Reply:
Yes, this is something interesting I realized lately when bootstrapping scanners: Windows requires the *absolute* path to the batch file here. This works for me:

    private fun gradleWrapper(version: String) {
        println("Installing Gradle wrapper version $version.")

        // When calling Windows batch files directly (without passing them to "cmd" as an argument), Windows requires
        // the absolute path to the batch file to be passed to the underlying ProcessBuilder for some reason.
        val wrapperAbsolutePath = File(projectDir, Gradle.wrapper).absolutePath

        ProcessCapture(projectDir, wrapperAbsolutePath, "wrapper", "--gradle-version=$version",
                "--distribution-type=ALL").requireSuccess()
    }

Reply:
Thanks, I have updated the PR.

Reply:
Still failing, I'm trying now if `projectDir.absoluteFile` helps.